hw/bsp/apollo3_evb/bsp.yml

```diff
@@ -38,7 +38,7 @@ bsp.flash_map:
         # System areas.
         FLASH_AREA_BOOTLOADER:
             device: 0
-            offset: 0x000c0000
+            offset: 0x0000c000
```

Comment:
This probably explains why I always have to flash my bootloader with jflash lite rather than mynewt 🤦

Reply:
Possibly.  :-)